src/SAWScript/Builtins.hs

```diff
@@ -1223,18 +1222,22 @@ tailPrim [] = fail "tail: empty list"
 tailPrim (_ : xs) = return xs
 
 parseCore :: String -> TopLevel Term
-parseCore input = do
```

Comment:
in general, don't mix code reformatting with other changes, it makes it harder to review